export/export.go

```diff
@@ -86,36 +89,78 @@ func NewResourcesService(client clientset.Interface, pods PodService, nodes Node
 
 // Get all resources from each service.
 func (s *Service) get(ctx context.Context) (*Resources, error) {
-	pods, err := s.podService.List(ctx)
-	if err != nil {
-		return nil, xerrors.Errorf("call list pods: %w", err)
-	}
-	nodes, err := s.nodeService.List(ctx)
-	if err != nil {
-		return nil, xerrors.Errorf("call list nodes: %w", err)
-	}
-	pvs, err := s.pvService.List(ctx)
-	if err != nil {
-		return nil, xerrors.Errorf("call list PersistentVolumes: %w", err)
-	}
-	pvcs, err := s.pvcService.List(ctx)
-	if err != nil {
-		return nil, xerrors.Errorf("call list PersistentVolumeClaims: %w", err)
-	}
-	scs, err := s.storageClassService.List(ctx)
-	if err != nil {
-		return nil, xerrors.Errorf("to call list storageClasses")
+	getCtx := context.Background()
+	g, _ := errgroup.WithContext(getCtx)
+	sem := semaphore.NewWeighted(int64(runtime.GOMAXPROCS(0)))
+	resources := Resources{}
+
+	sem.Acquire(getCtx, 1)
```

Comment:
get lint error with
```
export/export.go:97:13: Error return value of `sem.Acquire` is not checked (errcheck)
	sem.Acquire(getCtx, 1)
```
Should I check in if-statement?

Reply:
Yes.